helmfile.d/snippets/defaults.yaml

```diff
@@ -625,6 +625,12 @@ environments:
               limits:
                 cpu: 1000m
                 memory: 512Mi
+        databases:
+          keycloak:
+            useOtomiDB: true
+            imported: false
+            size: "1Gi"
+            replicas: 2
```

Comment:
Let's make replicas 1 by default, so users that evaluate Otomi by default does not consume too many resources 
replicas: 1